components-compose/src/main/java/uk/gov/hmrc/components/compose/atom/bullet/BulletedTextView.kt

```diff
@@ -20,23 +20,29 @@ import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.padding
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.clearAndSetSemantics
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import uk.gov.hmrc.components.compose.R
```

Comment:
I think we can clean up some of these imports that are no longer needed

Reply:
Oh no! I need to review my Android studio setting, previously these unused imports were removed automatically.